tests/src/common/WskCli.java

```diff
@@ -380,7 +394,7 @@ public String createPackage(int expectedCode, String name, Map<String, String> p
             op = "update";
         }
 
-        String[] cmd = { "package", op, "--auth", authKey, name };
+        String[] cmd = { "-i", "package", op, "--auth", authKey, name };
```

Comment:
having looked at so many `-i` - why not just add it once on the CLI call out where the edge host is applied?